llama-index-core/llama_index/core/agent/runner/base.py

```diff
@@ -355,6 +364,24 @@ def get_completed_steps(self, task_id: str, **kwargs: Any) -> List[TaskStepOutpu
         """Get completed steps."""
         return self.state.get_completed_steps(task_id)
 
+    def get_task_output(self, task_id: str, **kwargs: Any) -> TaskStepOutput:
+        """Get task output."""
+        completed_steps = self.get_completed_steps(task_id)
+        if len(completed_steps) == 0:
+            raise ValueError(f"No completed steps for task_id: {task_id}")
+        return completed_steps[-1]
```

Comment:
is it possible to have more than one item in this list?

Reply:
yes, but the output of a task is typically the last step (i.e. the step flagged as is_done)

Reply:
Right! Makes sense